src/models/Project.ts

```diff
@@ -25,6 +26,11 @@ export interface Project {
    */
   readonly dependencies: string[];
 
+  /**
+   * Graph of project dependencies
+   */
+  readonly graph: ProjectGraph;
```

Comment:
Find a way to separate this from the interface used by processors so they don't all need to worry about it.